pallet-chainlink-feed/src/lib.rs

```diff
@@ -737,9 +738,15 @@ pub mod pallet {
 		/// - Removes the details for the previous round if it was superseded.
 		///
 		/// Limited to the oracles of a feed.
-		#[pallet::weight(T::WeightInfo::submit_opening_round_answers().max(
-		T::WeightInfo::submit_closing_answer(T::OracleCountLimit::get())
-		))]
+		#[pallet::weight(
+            SubmitWeight {
+                weight: T::WeightInfo::submit_opening_round_answers().max(
+		            T::WeightInfo::submit_closing_answer(T::OracleCountLimit::get())
+                ),
+                origin,
```

Comment:
https://github.com/paritytech/substrate/issues/6316

Reply:
Could not use `origin` which is a parameter of `ensure_valid_round` in the weight macro